students/ericstreit/lessons/lesson09/assignment/database.py

```diff
@@ -0,0 +1,238 @@
+                 # yup! you were right!
+                 # CONVERT ROW DATA TO DICT
+                record_count = 0
+                error_count = 0
+                for row in products_contents:
+                     # this could be made into a func couldn't it?
+                    logging.info(f'IMPORTING PRODUCTS DATA')
+                    convert_to_dict = {'product_id': row[0],
+                                       'description': row[1],
+                                       'product_type': row[2],
+                                       'quantity_available': row[3]}
+                    try:
+                        result = products.insert_one(convert_to_dict)
+                        record_count += 1
+                    except Exception as e:
+                        logging.error(f'Error adding {convert_to_dict} to the products collection!')
+                        error_count += 1
+                    # who needs comments when your logging explains it all LOL
+                    logging.info(f'Adding {convert_to_dict} to the products collection')
+                for row in customers_contents:
+                    logging.info(f'IMPORTING CUSTOMERS DATA')
+                    convert_to_dict = {'user_id': row[0],
+                                       'name': row[1],
+                                       'address': row[2],
+                                       'zip_code': row[3],
+                                       'phone_number': row[4],
+                                       'email': row[5]}
+                    try:
+                        result = customers.insert_one(convert_to_dict)
+                        logging.info(f'Adding {convert_to_dict} to the customers collection')
+                        record_count += 1
+                    except Exception as e:
+                        logging.error(f'Error adding {convert_to_dict} to the customers collection!')
+                        error_count += 1
+                for row in rentals_contents:
+                    logging.info(f'IMPORTING RENTALS DATA')
+                    convert_to_dict = {'product_id': row[0],
+                                       'user_id': row[1]}
+                    try:
+                        result = rentals.insert_one(convert_to_dict)
+                        logging.info(f'Adding {convert_to_dict} to the rentals collection')
+                        record_count += 1
+                    except Exception as e:
+                        logging.error(f'Error adding {convert_to_dict} to the customers collection!')
+                        error_count += 1
+
+            print(f' total number of records added: {record_count}')
+            print(f' total number of documents in collection: {products.collection.estimated_data_count()}')
+        except Exception as e:
+            logging.error(f'Hm, we had an error here?')
+            logging.error(e)
+
+        return (record_count, error_count)
```

Comment:
Lol, now you really have to break up this function, if you are going to compare times of loading the different files.